lib/aux_js.cpp

```diff
@@ -334,6 +334,20 @@ void setKeyboardListeningElementId(const std::string & id)
    SDL_SetHint(SDL_HINT_EMSCRIPTEN_KEYBOARD_ELEMENT, id.c_str());
 }
 
+void processKeys(const std::string & keys)
+{
+  CallKeySequence(keys.c_str());
```

Comment:
Same here with `SendKeySequence()` instead of `CallKeySequence()`?

Reply:
I was following something @tzanio added in https://github.com/GLVis/glvis/commit/fe28c06c8eab4b899a5c0e33cd5f68c882aba035 , and I think we want `CallKeySequence` so that the keys are processed in a batch, otherwise you will visually see each applied in sequence instead of just the result.

Reply:
I think this ☝️ is right. For the web visualization `CallKeySequence` is better